Sources/PostgresNIO/New/PostgresCodable.swift

```diff
@@ -3,30 +3,69 @@ import class Foundation.JSONEncoder
 import class Foundation.JSONDecoder
 
 /// A type that can encode itself to a postgres wire binary representation.
-public protocol PostgresEncodable {
+public protocol PostgresEncodable: PostgresDynamicTypeEncodable {
     // TODO: Rename to `PostgresThrowingEncodable` with next major release
 
-    /// identifies the data type that we will encode into `byteBuffer` in `encode`
+    /// Identifies the data type that we will encode into `byteBuffer` in `encode`
     static var psqlType: PostgresDataType { get }
 
-    /// identifies the postgres format that is used to encode the value into `byteBuffer` in `encode`
+    /// Identifies the postgres format that is used to encode the value into `byteBuffer` in `encode`
     static var psqlFormat: PostgresFormat { get }
 
     /// Encode the entity into the `byteBuffer` in Postgres binary format, without setting
     /// the byte count. This method is called from the ``PostgresBindings``.
-    func encode<JSONEncoder: PostgresJSONEncoder>(into byteBuffer: inout ByteBuffer, context: PostgresEncodingContext<JSONEncoder>) throws
+    func encode<JSONEncoder: PostgresJSONEncoder>(
+        into byteBuffer: inout ByteBuffer,
+        context: PostgresEncodingContext<JSONEncoder>
+    ) throws
```

Comment:
This method is inherited, isn't it?

Reply:
Yep, also removed it from `PostgresNonThrowingEncodable` since that inherits `encode` from `PostgresDynamicTypeEncodable` now ([1801353](https://github.com/vapor/postgres-nio/pull/365/commits/18013530ad476150106ae4e26e14863a888988b3))